src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -670,11 +670,11 @@ protected override void Generate(RenameTableOperation operation, IModel model, M
 
         protected override void IndexOptions(CreateIndexOperation operation, IModel model, MigrationCommandListBuilder builder)
         {
-            if (operation[NpgsqlAnnotationNames.IndexInclude] is string[] includeProperties && includeProperties.Length > 0)
+            if (operation[NpgsqlAnnotationNames.IndexInclude] is string[] includeColumns && includeColumns.Length > 0)
```

Comment:
And here you're requesting an array. It's fine if you're expecting that only an array should be here and not `IReadOnlyList<string>`.

Reply:
Yeah, I think it's OK to expect an array as NpgsqlMigrationsAnnotationProvider is the only thing the produces it.